examples/powershell/vro/vro-secrets.json

```diff
@@ -0,0 +1,8 @@
+{
+    "VRO_SERVER" : "vro.cpbu.corp",
+    "VRO_USERNAME": "william@cpbu.corp",
```

Comment:
Probably better to use a generic name here, just so people don't think you leaked information :)

Reply:
:)